standards/README.md

```diff
@@ -17,3 +18,4 @@ This folder contains all current standards for software development.
 - [Security standards](security_standards.md)
 - [TSQL and SQL Server database standards](tsql_and_sqldb_standards.md)
 - [Version control standards](version_control_standards.md)
+- [Docker standards](docker_standards.md)
```

Comment:
This line should be removed as it has been replaced by "Container standards"

Reply:
Updated